### PR TITLE
Event category_delete added to category bulk_delete mutation

### PR DESCRIPTION
**Pull request by SzymJ**
I want to merge this change because it adds sending `category_delete` webhook event to category `bulk_delete` mutation.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-6567 Add category_delete webhook event to bulk_delete mutation](https://app.clickup.com/t/2549495/SALEOR-6567)

**Review by maarcingebala — APPROVED** [review]

File: saleor/graphql/product/bulk_mutations/products.py [hunk 1]
```diff
@@ -85,6 +85,9 @@ class Meta:
     def bulk_action(cls, info, queryset):
         delete_categories(queryset.values_list("pk", flat=True), info.context.plugins)
 
+        for instance in queryset:
+            info.context.plugins.category_deleted(instance)
```
Comment (korycins) [inline saleor/graphql/product/bulk_mutations/products.py:89]: Keep in mind, that your `instances` here are already deleted, so some part of data could not be accessible here.

**Review by IKarbowiak — APPROVED** [review]

File: saleor/graphql/product/bulk_mutations/products.py (hunk 1 above)
Comment (korycins) [reply saleor/graphql/product/bulk_mutations/products.py:89]: Also I just quickly look on the tests like `saleor/plugins/webhook/tests/subscription_webhooks/test_create_deliveries_for_subscription.py::test_category_deleted` 
I think that we could have here a fail success. 
You're testing building a logic based on object **that exists in database**. But in real evn, this logic will be executed on object that was already deleted, so some fields could not be accesible.

Comment (korycins) [reply saleor/graphql/product/bulk_mutations/products.py:89]: I can confirm. The subscription from test `test_category_deleted` returns `base64(Category:None)` as an ID, when object doesn't exists.

Comment (SzymJ) [reply saleor/graphql/product/bulk_mutations/products.py]: I moved `info.context.plugins.category_deleted(instance)` to `delete_categories` function and before I created instances list before `.delete()`.
I updated `test_create_deliveries_for_subscription.py::test_category_deleted` test to use the same flow and I added assert to check if `global_id` is not `None`

File: saleor/plugins/webhook/tests/subscription_webhooks/test_create_deliveries_for_subscription.py [hunk 2]
```diff
@@ -43,14 +44,22 @@ def test_category_updated(category, subscription_category_updated_webhook):
...
 
     # when
-    deliveries = create_deliveries_for_subscriptions(event_type, category, webhooks)
+    deliveries = create_deliveries_for_subscriptions(
+        event_type, category_instances[0], webhooks
+    )
 
     # then
     expected_payload = json.dumps({"category": {"id": category_id}, "meta": None})
+    assert graphene.Node.from_global_id(category_id)[1] != "None"
```
Comment (korycins) [inline saleor/plugins/webhook/tests/subscription_webhooks/test_create_deliveries_for_subscription.py:62]: You could do that without converting `category_id` from global ID. You could just check if DB's id is not None. 
Something like 👇 
```
category_instances = [cat for cat in category_query]
db_category_id = category_instances[0]
...
# then
assert db_category_id
expected_payload
...
```

**Review by korycins — APPROVED** [review]

**Pull request merged**